ConsentViewController/Classes/SPMessageUIDelegate.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  MessageUIDelegate.swift
+//  MessageUIDelegateSpy.swift
```

Comment:
@andresilveirah you haven't renamed the class or a filename, this might be a redundant change

Reply:
I see `MessageUIDelegateSpy: SPMessageUIDelegate` usage, in the sake of consistency I suggest renaming public protocol `SPMessageUIDelegate` or filename

Reply:
You have a point @Nevazhnovu , I'll just revert that comment back to `SPMessageUIDelegate`